pkg/client/unversioned/deployment.go

```diff
@@ -82,6 +83,13 @@ func (c *deployments) Create(deployment *extensions.Deployment) (result *extensi
 	return
 }
 
+// Patch takes the partial representation of a deployment and updates it. Returns the server's representation of the deployment, and an error, if there is any.
+func (c *deployments) Patch(name string, pt api.PatchType, data []byte, subresources ...string) (result *extensions.Deployment, err error) {
+	result = &extensions.Deployment{}
+	err = c.client.Patch(api.StrategicMergePatchType).Namespace(c.ns).Resource("deployments").SubResource(subresources...).Name(name).Body(data).Do().Into(result)
+	return
+}
+
```

Comment:
Yes. Update fails on update conflicts whereas Patch doesn't. Since the pause field is supposed to be manipulated only via `pause` and `resume` (and `delete`) then it makes sense to patch in these commands.